gatsby/gatsby-config.js

```diff
@@ -98,16 +99,30 @@ const getPlugins = () => {
     {
       resolve: `gatsby-source-filesystem`,
       options: {
-        name: `docs-gatsby  `,
+        name: `docs-gatsby`,
         path: `${__dirname}/src/docs`
       }
     },
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `includes`,
+        path: `${__dirname}/src/includes`
+      }
+    },
     {
       resolve: `gatsby-source-filesystem`,
       options: {
         name: `pages`,
         path: `${__dirname}/src/pages`
       }
+    },
+    {
```

Comment:
this makes all yaml files available as a graphql collection. you can see the platform example in the PlatformContent component.